android/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ include_directories(SYSTEM
 )
 
 # Disable some compiler warnings (the lazy way out)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-double-promotion -Wno-sign-conversion")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-double-promotion -Wno-sign-conversion -Wno-unused-command-line-argument -s")
```

Comment:
`-s` strips relocations as well, is that intended?

Reply:
Doesn't seem to make a difference as far as I can see.

Reply:
If it works, then no problem.